components/button/demo/button.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+		<meta charset="UTF-8">
+		<link rel="stylesheet" href="../../demo/styles.css" type="text/css">
+		<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script type="module">
+			import '../../demo/demo-page.js';
+			import '../button.js';
+		</script>
+	</head>
+	<body unresolved>
+
+		<d2l-demo-page title="d2l-button">
+			<h3>Button</h3>
+			<d2l-demo-snippet>
+				<d2l-button>Normal Button</d2l-button>
+			</d2l-demo-snippet>
+			<h3>Button (Primary)</h3>
+			<d2l-demo-snippet>
+				<d2l-button primary="">Primary Button</d2l-button>
```

Comment:
Nit: these boolean attributes can just be `<d2l-button primary disabled>` -- no need to set them to empty string anymore (I think that was a Polymer 1 -> auto-conversion ugly artifact).